specification/commerce/resource-manager/Microsoft.Commerce/preview/2015-06-01-preview/commerce.json

```diff
@@ -137,6 +143,12 @@
               "$ref": "#/definitions/ResourceRateCardInfo"
             }
           },
+          "202": {
+            "description": "Response indicating that a request has been accepted for processing. However, the processing has not been completed.",
```

Comment:
You should not make any change to the ratecard block of the swagger, ratecard is a different service and I don't think they support 202 response code